Library/Homebrew/build.rb

```diff
@@ -105,7 +105,8 @@ def install
 
     formula.extend(Debrew::Formula) if ARGV.debug?
 
-    formula.brew do
+    formula.brew do |f, stg|
```

Comment:
Right, I meant, I was avoiding using that name for these `|<foo>|` block parameters, because I wanted to use the same parameter name for all of them, and in some of these blocks, a call to a `stage` method might actually occur, so we need to avoid the local variable masking the same-named method. And since `stage` is a method name used in this functional area, I thought it would be clearer to keep separate names rather than rely on the type precedence to sort things out.

> I personally prefer stage_context, though staging_context sounds more correct to my non-native understanding of the English language.

"Staging context" is how you'd say it in spoken conversation, but I think `stage_context` works better as a variable, too. If there were several things related to a `stage`, I'd prefix them all with `stage_`, instead of switching between `stage_`, `staged_` and `staging_`.